### PR TITLE
feat: Quick polls

### PR DESCRIPTION
**Pull request by hardikbhatia777**
/quickpoll - Creates a quick poll

### CODE REVIEW

File: commands/Poll.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		const room: IRoom = context.getRoom();
+
+		const args = context.getArguments();
+
+		if (args.length < 2) {
+			await sendMessageToRoom(room, modify, user ?? author, t('please_provide_both_arguments'));
+			return;
+		}
+
+		const time = args[0];
```
Comment (scuciatto) [inline commands/Poll.ts:41]: Wouldn't it be good to add error handling for parsing time? This would improve robustness.

File: commands/Poll.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			responses: {
+				yes: [],
+				no: [],
+			},
+		};
+
+		const assoc = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, uuid);
+		await persis.createWithAssociation(pollData, assoc);
+
+		const blockBuilder = modify.getCreator().getBlockBuilder();
```
Comment (scuciatto) [inline commands/Poll.ts:64]: Blockbuilder is deprecated. We should use UIKit instead. We have discussed it here: https://open.rocket.chat/group/RLzgTEq2ngE2cMnMF?msg=tN8Sc895GFpHmkD4u

File: commands/Poll.ts (hunk 1 above)
Comment (scuciatto) [inline commands/Poll.ts:41]: Can we add error handling for parsing time? This would improve robustness.

**Review by scuciatto — CHANGES_REQUESTED** [review]

File: commands/Poll.ts (hunk 1 above)
Comment (hardikbhatia777) [reply commands/Poll.ts]: Added checks to ensure it's a valid (non-zero and below 10000) integer.

File: commands/Poll.ts (hunk 2 above)
Comment (hardikbhatia777) [reply commands/Poll.ts]: Done :)

**Review by scuciatto — APPROVED** [review]

**Pull request merged**